modules/reactor/test/tests/meshgenerators/hexagon_mesh_trimmer/assembly_trim.i

```diff
@@ -0,0 +1,55 @@
+[Mesh]
+  final_generator = trim
+  [hex_1]
+    type = PolygonConcentricCircleMeshGenerator
+    num_sides = 6
+    num_sectors_per_side = '4 4 4 4 4 4'
+    background_intervals = 2
+    ring_radii = 4.0
+    ring_intervals = 2
```

Comment:
```suggestion
    num_sectors_per_side = '1 1 1 1 1 1'
    background_intervals = 1
    ring_radii = 4.0
    ring_intervals = 1
```

Reply:
2 is the minimum value for `num_sectors_per_side`